packages/deployment/app/config.yml

```diff
@@ -100,14 +100,6 @@ default:
     targetValue: 30
     scaleInCooldown: 0
     scaleOutCooldown: 0
-  GranulesTable:
```

Comment:
Removed this configuration because I don't know if we actually want to enable auto scaling for these tables by default